### ISSUE TITLE
Observers not working on child components with data passed from parent (see fiddle)

### ISSUE THREAD

**Issue by alexmakeev** [ractivejs/ractive#2233]
When you create child component and pass data to it, the child can't observe data with wildcard expression (observers work only once, on init)
See fiddle for example: http://jsfiddle.net/alexmak/6xn6stsu/4/

**Comment by fskreuz** [comment]
Simplified the case here: http://jsfiddle.net/6xn6stsu/5/

```
<div id="main"></div>
<script type="text/template" id="main_template">
    <setter items="{{items}}" />
</script>

<script type="text/template" id="setter_template">
	<div>
		<input type="checkbox" checked="{{items.0.val}}" />
		<input type="checkbox" checked="{{items.1.val}}" />
		<input type="checkbox" checked="{{items.2.val}}" />
        Click checkboxes to trigger observers. See console
	</div>
</script>

var SetterComponent = Ractive.extend({
    template: '#setter_template',
    oninit: function () {
        this.observe("items.*.val", function (nv, ov, kp) { console.log("CHILD WILDCARD:", nv, ov, kp) });        
        this.observe("items.0.val items.1.val items.2.val", function (nv, ov, kp) { console.log("CHILD KEYED:", nv, ov, kp) });
    }
});

var app = new Ractive({
    el: '#main',
    template: '#main_template',
    data: {
        items: [{ val: true }, { val: true }, { val: true }]
    },
    components: {
        setter: SetterComponent
    },
    oninit: function () {
        this.observe("items.*.val", function (nv, ov, kp) {
            console.log('PARENT:', nv, ov, kp)
        })
    }
});
```

Seems like the wildcard for descendants isn't working. Should run by @martypdx because I remember a discussion about observers, change handling and data ownership.

**Comment by dagnelies** [comment]
Might be related to #1988

**Comment by Madgvox** [comment]
As this has been fixed on 0.8-edge, I am going to close this issue.
